packages/postgres-database/src/simcore_postgres_database/models/projects_to_folders.py

```diff
@@ -0,0 +1,49 @@
+import sqlalchemy as sa
+
+from ._common import (
+    column_created_datetime,
+    column_modified_datetime,
+    register_modified_datetime_auto_update_trigger,
+)
+from .base import metadata
+from .folders_v2 import folders_v2
+
+projects_to_folders = sa.Table(
```

Comment:
MINOR: for consistency if the table is projects_to_folders, add then first the column project and then folders and not the other way around

Reply:
improved